src/common/Post/components/Stream/index.tsx

```diff
@@ -1,21 +1,17 @@
-import React, { useRef, useEffect, useState } from "react";
+import { useRef, useEffect, useState } from "react";
 import TipRibbon from "../TipRibbon";
 import "./css/index.scoped.css";
 import videojs from "video.js";
 
 const Stream = ({
-  id,
   item,
-  index,
-  postId,
   tipValue,
   tipCounter,
   hideRibbon,
   width,
-  timeout = 10000
 }) => {
   const playerDOM = useRef(null);
-  const [isLive, setIsLive] = useState(false);
+  const [isLive] = useState(false);
```

Comment:
if setIsLive is unused then `isLive` is probably _always_ false, can you get rid of that too?

Reply:
done